lib/browser/utils.ts

```diff
@@ -0,0 +1,38 @@
+import { EventEmitter } from 'events'
+
+/**
+* Creates a lazy instance of modules that can't be required before the
+* app 'ready' event by returning a proxy object to mitigate side effects
+* on 'require'
+*
+* @param {Function} creator - Function that creates a new module instance
+* @param {Object} holder - the object holding the module prototype
+* @param {Boolean} isEventEmitter - whether or not the module is an EventEmitter
+* @returns {Object} - a proxy object for the
+*/
+
+export function createLazyInstance (
```

Comment:
why not use an ES6 Proxy?

Reply:
probably not unfortunately, the intricacies of doing that correctly would introduce far more space for potential issues

Reply:
The `remote` module does not work very well with ES6 Proxy objects.